config/locale/de.json

```diff
@@ -5,13 +5,13 @@
     "404 No monsters found":"Fehler: Kein gültiges Pokémon angegeben.",
     "404 No raid egg levels found":"Fehler: Kein Raid-Ei-Level angegeben.",
     "404 No valid invasion types found":"Fehler: Es wurde kein gültiger Typ einer Team-GO-Rocket-Invasion angegeben.",
-    "404 No valid quests found":"Fehler: Kein gültiges Quest angegeben.",
+    "404 No valid quests found":"Fehler: Keine gültige Quest angegeben.",
     "404 no valid tracks found":"Fehler: Kein gültiger Raid-Boss oder gültiges Raid-Level angegeben.",
     "add":"hinzufügen",
     "Added areas:":"Hinzugefügte Gebiete:",
     "Adventure Hat 2020":"Forscher",
-    "all items":"alle items",
-    "all pokemon":"alle pokemon",
+    "all items":"alle Items",
+    "all pokemon":"alle Pokemon",
```

Comment:
Does this actually work? Poracle forces commands to lower case and uses reverse translation so I have a suspicion this might prevent these commands from executing?

Reply:
I will remove that change, sorry..

Reply:
The test command would be equivalent of !quest all_pokemon (note underscore used to make a keyword with spaces)